tests/index.spec.ts

```diff
@@ -33,13 +33,14 @@ describe('FilePath', () => {
 
   it('normalizes path', () => {
     const path = new FilePath('this/is/../is/../is/./a/full/path.jpeg');
-
     expect(path.folders).toEqual(['this', 'is', '..', 'is', '..', 'is', '.', 'a', 'full']);
+
+    path.dir = '/this/is/a/half';
```

Comment:
There is a test for [dir already](https://github.com/autovance/filepath/pull/3/files#diff-e1e998c28731133ec4ebe0fa428bfad3R75).
Setting it here overwrites the `folders` so the normalization isn't actually tested, since the new dir is already normalized.

Reply:
Your link doesn't go anywhere, I'll remove this, it was part of me playing around.